### ISSUE TITLE
crypto/cversion.c compile error around OPENSSL_VERSION_TEXT

### ISSUE THREAD

**Issue by gjasny** [openssl/openssl#11618]
While building 3.0.0-alpha1 I noticed the following compilation error:

```
$ untar the 3.0.0-alpha1 tarball
$ ./config
$ make -j8
/Applications/Xcode.app/Contents/Developer/usr/bin/make depend && /Applications/Xcode.app/Contents/Developer/usr/bin/make _build_sw
cc  -Icrypto -I. -Iinclude -Iproviders/common/include -Iproviders/implementations/include -Icrypto/include  -DAES_ASM -DBSAES_ASM -DCMLL_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -arch x86_64 -O3 -Wall -DL_ENDIAN -DOPENSSL_BUILDING_OPENSSL -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -D_REENTRANT -DNDEBUG  -MMD -MF crypto/libcrypto-lib-cversion.d.tmp -MT crypto/libcrypto-lib-cversion.o -c -o crypto/libcrypto-lib-cversion.o crypto/cversion.c
crypto/cversion.c:50:16: error: expected ';' after return statement
        return OPENSSL_VERSION_TEXT;
               ^
include/openssl/opensslv.h:91:54: note: expanded from macro 'OPENSSL_VERSION_TEXT'
# define OPENSSL_VERSION_TEXT "OpenSSL 3.0.0-alpha1 "23 Apr 2020""
                                                     ^
1 error generated.
```

I could reproduce it by compiling 05feb0a0f1fecb6839888bb7590fb92be70d8d3c but not in the following d38997af165324b1ec0bbc0c6694daf7f503f080 commit. My suspicion is that the release date is wrongly quoted: https://github.com/openssl/openssl/blob/05feb0a0f1fecb6839888bb7590fb92be70d8d3c/VERSION#L6

Thanks,
Gregor

**Comment by mattcaswell** [comment]
That's....annoying. Looks like the release script itself broke the compilation.

**Comment by mattcaswell** [comment]
Temporary workaround for the moment is to remove the quotes on the RELEASE_DATE line in the VERSION file:

```` diff
diff --git a/VERSION b/VERSION
index b7529b46be..bfcd4fc7e2 100644
--- a/VERSION
+++ b/VERSION
@@ -3,5 +3,5 @@ MINOR=0
 PATCH=0
 PRE_RELEASE_TAG=alpha1
 BUILD_METADATA=
-RELEASE_DATE="23 Apr 2020"
+RELEASE_DATE=23 Apr 2020
 SHLIB_VERSION=3
````

**Comment by mattcaswell** [comment]
I added some text about this in the release notes on the wiki:
https://wiki.openssl.org/index.php/OpenSSL_3.0

**Comment by levitte** [comment]
Real fix in #11624

**Comment by tavrez** [comment]
Just wanted to add a note that touching VERSION file won't recreate include/openssl/opensslv.h from opensslv.h.in file, delete the file to make "make" recreate it(need a fix in generate makefile)
